lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -275,29 +281,16 @@ sub _show {
             scenario                => $job->scenario_name,
             worker                  => $worker,
             assigned_worker         => $job->assigned_worker,
+            resultfiles             => $resultfiles,
+            ulogs                   => $ulogs,
+            show_details            => ($test_modules || grep { /autoinst-log.txt/ } @$resultfiles),
```

Comment:
We also need to show the details tab for any running test, even though no test modules have been uploaded yet. (At least when I remember the JavaScript code for running tests correctly.)